### PR TITLE
Add Cell.hasHttpSess(iden)

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: synapse/lib/cell.py [hunk 1]
```diff
@@ -2595,6 +2595,9 @@ def getUserName(self, iden, defv='<unknown>'):
             return defv
         return user.name
 
+    async def hasHttpSess(self, iden):
+        return bool(await self.getHttpSessDict(iden))
```
Comment (invisig0th) [inline synapse/lib/cell.py:2599]: A sess dict that is empty would return False here. Is that possible/intended?

**Review by invisig0th — CHANGES_REQUESTED** [review]
What's the use case for the API?

File: synapse/lib/cell.py (hunk 1 above)
Comment (vEpiphyte) [reply synapse/lib/cell.py]: It shouldn't be possible unless someone has popped the `created` value out of the session. I've added a method on the guidstor to do an existence check and am using that.

**Comment by vEpiphyte** [comment]
Use case is for checking if an existing guid is present, in order to prevent round-tripping to the http handler authcell.

**Review by invisig0th — APPROVED** [review]

**Pull request merged**